### DISCUSSION TITLE
Configuration options?

### DISCUSSION THREAD

**Discussion by S1M0NH** [cherrypy/cheroot#79] [Q&A] [answered]
* **I'm submitting a ...**
[ ] bug report
[x] feature request
[ ] question about the decisions made in the repository

* **Do you want to request a *feature* or report a *bug*?**

Hello. I'm using cheroot with bottle and am wondering if options can be passed in -- such as force https, serve multiple sites etc. I'm using cheroot on it's own (no apache or nginx) and would prefer to keep it that way unless not possible.

* **What is the current behavior?**

* **If the current behavior is a bug, please provide the steps to reproduce and if possible a screenshots and logs of the problem. If you can, show us your code.**

* **What is the expected behavior?**

* **What is the motivation / use case for changing the behavior?**

* **Please tell us about your environment:**

- Python version: 2.7.9
- Cheroot version: 6.0.0
- CherryPy version: X.X.X (if applicable)
- OS: LInux 
- Browser: [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, e.g. stackoverflow, gitter, etc.)

**Comment by webknjaz** [comment, marked as answer]
This should answer your question https://github.com/cherrypy/cheroot/blob/7268068/cheroot/wsgi.py#L55-L70

**Comment by webknjaz** [comment]
I'm assuming you've found your answer. Closing.
